packs/src/items/equipment/robe-of-useful-items.json

```diff
@@ -5,7 +5,7 @@
+      "value": "<p><em>Wondrous item</em></p>\n<p>This robe has cloth patches of various shapes and colors covering it. While wearing the robe, you can use an action to detach one of the patches, causing it to become the object or creature it represents. Once the last patch is removed, the robe becomes an ordinary garment.</p>\n<p>The robe has the following patches:</p>\n<ul>\n<li>x2 Dagger patches</li>\n<li>x2 Bullseye lantern (filled and lit) patches</li>\n<li>x2 Steel mirror patches</li>\n<li>x2 10-foot pole patches</li>\n<li>x2 Hempen rope (50 feet, coiled) patches</li>\n<li>x2 Sack patches</li>\n</ul>\n<p>In addition, the robe has [[4d4]] other patches. The GM chooses the patches or determines them randomly from the @Compendium[dnd5e.tables.CTGxcd51szehUGBn]{Robe of Useful Items} table.</p>\n<p><em>Foundry note: the Other Formula has been set to calculate the 4d4 patches if required.</em></p>\n<p><em>Additionally, it is recommended to delete items here as necessary to track patches left.</em></p>\n<p><em>The price guide has been calculated at 5*item for the maximum number of items possible.  Please adjust as necessary to reflect the number of patches and the DM's world economy.</em></p>\n<p> </p>",
```

Comment:
The inline roll for number of patches here should be a deferred roll (`[[/r 4d4]]`) otherwise the value changes every time you open up the item description.

Also, since the 4d4 is in the description you can probably remove it from the Other Formula field and the description's Foundry Note.

```suggestion
      "value": "<p><em>Wondrous item</em></p>\n<p>This robe has cloth patches of various shapes and colors covering it. While wearing the robe, you can use an action to detach one of the patches, causing it to become the object or creature it represents. Once the last patch is removed, the robe becomes an ordinary garment.</p>\n<p>The robe has the following patches:</p>\n<ul>\n<li>x2 Dagger patches</li>\n<li>x2 Bullseye lantern (filled and lit) patches</li>\n<li>x2 Steel mirror patches</li>\n<li>x2 10-foot pole patches</li>\n<li>x2 Hempen rope (50 feet, coiled) patches</li>\n<li>x2 Sack patches</li>\n</ul>\n<p>In addition, the robe has [[/r 4d4]] other patches. The GM chooses the patches or determines them randomly from the @Compendium[dnd5e.tables.CTGxcd51szehUGBn]{Robe of Useful Items} table.</p>\n<p><em>Foundry note: It is recommended to delete items here as necessary to track patches left.</em></p>\n<p><em>The price guide has been calculated at 5*item for the maximum number of items possible.  Please adjust as necessary to reflect the number of patches and the DM's world economy.</em></p>\n<p> </p>",
```